### PR TITLE
Tag with lastest

### LINKED ISSUE #171: Stop tagging things with :master, tag with :latest
I've accidentally tested the wrong version locally because of this...

### PR DESCRIPTION
**Pull request by daixiang0**
Fix #171

Signed-off-by: Xiang Dai <764524258@qq.com>

### CODE REVIEW

File: production/README.md [hunk 1]
```diff
@@ -65,7 +65,7 @@ $ ./promtail -config.file=./cmd/promtail/promtail-local-config.yaml
 Grafana is Loki's UI, so you'll also want to run one of those:
 
 ```bash
-$ docker run -ti -p 3000:3000 -e "GF_EXPLORE_ENABLED=true" grafana/grafana:master
+$ docker run -ti -p 3000:3000 -e "GF_EXPLORE_ENABLED=true" grafana/grafana:latest
```
Comment (tomwilkie) [inline production/README.md]: This one needs to stay as `:master`, as thats what grafana pushes.

**Comment by daixiang0** [comment]
@tomwilkie Thanks for your review! I am sorry that i do not notice it in `Recent activity` from github.
Happy for your response!

**Comment by daixiang0** [comment]
@tomwilkie hi, please review this again, thanks!

**Comment by daixiang0** [comment]
@tomwilkie please have a look

**Comment by daixiang0** [comment]
@tomwilkie hi, please review this again, thanks!

**Comment by daixiang0** [comment]
@tomwilkie mind have a look?

**Comment by daixiang0** [comment]
any update?

**Comment by daixiang0** [comment]
@tomwilkie hi, mind have a look?

File: .circleci/config.yml [hunk 2]
```diff
@@ -130,7 +130,7 @@ jobs:
           name: Push Images
           command: |
             docker login -u "$DOCKER_USER" -p "$DOCKER_PASS" &&
-            make push-master
```
Comment (steven-sheehy) [inline .circleci/config.yml]: Still lastest

**Comment by daixiang0** [comment]
@steven-sheehy done

**Comment by daixiang0** [comment]
any update? @steven-sheehy

**Comment by steven-sheehy** [comment]
@daixiang0 LGTM, but you'll have to wait for a Loki dev to approve.

**Comment by daixiang0** [comment]
@tomwilkie mind have a look?

**Comment by daixiang0** [comment]
@davkal done, please have a second look.

**Pull request merged**